g_docs.go

```diff
@@ -509,16 +509,24 @@ func parserComments(comments *ast.CommentGroup, funcName, controllerName, pkgpat
 						typ = typ[2:]
 						isArray = true
 					}
+
 					if typ == "string" || typ == "number" || typ == "integer" || typ == "boolean" ||
-						typ == "array" || typ == "file" || typ == "enum" {
+						typ == "array" || typ == "file" {
 						paraType = typ
 					} else if sType, ok := basicTypes[typ]; ok {
 						typeFormat := strings.Split(sType, ":")
 						paraType = typeFormat[0]
 						paraFormat = typeFormat[1]
+					} else if typ == "enum" {
+						paraType = "string"
+						para.Enum = strings.Split(p[4], ",")
```

Comment:
OK I dont have any context on what's going on 😅  So just checking if the code is OK'ish.

I see the former code had a check of `if len(p) > 4 {` and then the splitting took place (too avoid a index out of bound access). Is it still necessary or is the fact that we are in `typ == "enum"` sufficient?

Reply:
that's right! because when adding `enum` we should always provide values for the enum to use. for example:
```
// @Param	testingDevice 	query	enum	false	ios,android
```
in this case it's `ios,android`. 

I've tested this against DOR `develop` branch, and it works fine afaics 👍 . I also just added more comments and the length-check to validate future params if we missed to add one. Addressed in https://github.com/zalora/bee/pull/9/commits/61572c1bc01e331672b0da752a2b25f9fac568bb